examples/DayPickerRangeControllerWrapper.jsx

```diff
@@ -31,6 +35,7 @@ const propTypes = forbidExtraProps({
   orientation: ScrollableOrientationShape,
   withPortal: PropTypes.bool,
   initialVisibleMonth: PropTypes.func,
+  range: RangeShape,
```

Comment:
offsetRange maybe? are we still using this?

Reply:
Replaced with startDateOffset/endDateOffset